src/model.ts

```diff
@@ -17,12 +17,15 @@ export async function getCommitCoverageRatio(
     { repo, rev }: Pick<ResolvedRootURI, 'repo' | 'rev'>,
     endpoint: Endpoint,
     sourcegraph: typeof import('sourcegraph')
-): Promise<number | undefined> {
+): Promise<number | null | undefined> {
     const data = await codecovGetCommitCoverage({
         ...codecovParamsForRepositoryCommit({ repo, rev }, sourcegraph),
         baseURL: endpoint.url,
         token: endpoint.token,
     })
+    if (data === null) {
+        return data
```

Comment:
should this be `return null` to be consistent with usage below?

Reply:
Yup.

```suggestion
        return null
```

Reply:
ok, how did you do that?
![image](https://user-images.githubusercontent.com/4213028/71685131-65239900-2d65-11ea-9729-8f64830800fe.png)
I've never seen this in a PR comment. so cool!

Reply:
I used GitHub suggestions:

![image](https://user-images.githubusercontent.com/1741180/71685239-b6cc2380-2d65-11ea-90e5-574d9d46b5ce.png)

It's a convenient way to make one-line fixes directly from the PR review UI, as a reviewer I use it a lot to suggest fixes that can be directly applied by author!

Reply:
yeah, this is awesome! thanks for showing!